pipeline/cloud/schemas/pipelines.py

```diff
@@ -131,6 +131,8 @@ class PipelineState(str, Enum):
     not_loaded = "not_loaded"
     loading = "loading"
     loaded = "loaded"
+    load_failed = "load_failed"
+    # This is really startup_failed
     failed = "failed"
```

Comment:
Under what condition would a pipeline be reported as `failed` and not `startup_failed`

Reply:
Sorry think this is confusing, the comment is intended for `failed`. Basically `load_failed` happens when we import the model, `failed` is our already in place state for an issue during pipeline startup. Hence I commented that it should really be `startup_failed`. I think we'll probably add some more failure modes here so worth refactoring at a later date.